### DISCUSSION TITLE
Known packages that need to run as a singleton

### DISCUSSION THREAD

**Discussion by XiaofengXie16** [module-federation/module-federation-examples#1633] [General]
Known packages that need to run as a singleton :
- react
- react-dom
- @datadog/browser-rum
- react-router-dom

Feel free to add more items in this discussion.

**Comment by AndreiSoroka** [comment]
frameworks:
vue
vuex
svelte
...
full popular list https://2020.stateofjs.com/en-US/technologies/front-end-frameworks/

**Comment by AndreiSoroka** [comment]
Design system, like primefaces or bootstrap. Need investigate and find all list

**Comment by ScriptedAlchemy** [comment]
Anything that uses context. Needs to be singletons. 

So theme providers. Routers, redux, emotion and other css in js
